Symfony/CS/Fixer/Symfony/PhpdocTypesFixer.php

```diff
@@ -24,26 +24,9 @@
      * @var string[]
      */
     private static $types = array(
-        'array',
-        'bool',
-        'boolean',
-        'callable',
-        'double',
-        'false',
-        'float',
-        'int',
-        'integer',
-        'mixed',
-        'null',
-        'object',
-        'real',
-        'resource',
-        'self',
-        'static',
-        'string',
-        'true',
-        'void',
-        '$this',
+        'array', 'bool', 'boolean', 'callable', 'double', 'false', 'float',
```

Comment:
Just changed it because we seem to do that style everywhere else. I can revert it though. Maybe we need a fixer to force every element onto its own newline?